test/codeu/chat/CreateInterestsTest.java

```diff
@@ -0,0 +1,5 @@
+package codeu.chat;
+
+public class CreateInterestsTest {
+
```

Comment:
It would be awesome to see tests, but like we discussed, we're getting down to the deadline. Maybe think about going back and adding tests if you get some time.